presto-docs/src/main/sphinx/functions/aggregate.rst

```diff
@@ -113,7 +113,29 @@ General Aggregate Functions
     for each input value. In addition to taking the input value, ``inputFunction``
     takes the current state, initially ``initialState``, and returns the new state.
     ``combineFunction`` will be invoked to combine two states into a new state.
-    The final state is returned::
+    The final state is returned. Throws an error if ``initialState`` is NULL.
```

Comment:
```suggestion
    The final state is returned. If ``initialState`` is NULL, an error is returned.
```

Reply:
I agree with you about active voice as a best practice for writing. I don't feel this is a big deal either and I am okay with your decision. 

To explain my thoughts here: I rephrased the sentence to emphasize the condition that the error happens when ``initialState`` is NULL. By presenting the condition first, the reader can identify if the rest of the sentence applies to their situation. 

Thanks for the discussion!